addons/l10n_it_edi/models/res_config_settings.py

```diff
@@ -62,7 +62,7 @@ def button_create_proxy_user(self):
         if not edi_identification:
             return
 
-        self.env['account_edi_proxy_client.user']._register_proxy_user(self.company_id, fattura_pa, edi_identification)
+        self.env['account_edi_proxy_client.user']._register_proxy_user(self.company_id, 'l10n_it_edi', edi_identification)
```

Comment:
We cannot change this unless we change this 😅  
![image](https://user-images.githubusercontent.com/1665365/228222087-d55d861a-03dd-4144-880c-4b2c010220a6.png)

Reply:
Or we can put `('edi_format_code', 'in', ('fattura_pa', 'l10n_it_edi'))` and have a transition period